nifi-marklogic-processors/src/main/java/org/apache/nifi/marklogic/processor/QueryRowsMarkLogic.java

```diff
@@ -24,85 +24,106 @@
 @CapabilityDescription("Executes a serialized Optic query plan and writes the returned rows to a FlowFile")
 public class QueryRowsMarkLogic extends AbstractMarkLogicProcessor {
 
-    public static final PropertyDescriptor PLAN = new PropertyDescriptor.Builder()
-        .name("Plan")
-        .displayName("Plan")
-        .description("A serialized Optic query plan; see https://docs.marklogic.com/guide/app-dev/OpticAPI#id_11208 for information on how to generate a plan")
-        .required(true)
-        .addValidator(Validator.VALID)
-        .expressionLanguageSupported(ExpressionLanguageScope.FLOWFILE_ATTRIBUTES)
-        .build();
-
-    public static final PropertyDescriptor MIMETYPE = new PropertyDescriptor.Builder()
-        .name("MIME Type")
-        .displayName("MIME Type")
-        .description("The MIME type to use when returning the rows")
-        .defaultValue("text/csv")
-        .required(true)
-        .addValidator(Validator.VALID)
-        .expressionLanguageSupported(ExpressionLanguageScope.FLOWFILE_ATTRIBUTES)
-        .build();
-
-    protected static final Relationship FAILURE = new Relationship.Builder().name("failure")
-        .description("FlowFiles that were not successfully processed are routed here").build();
-
-    protected static final Relationship SUCCESS = new Relationship.Builder().name("success")
-        .description("A FlowFile is routed here with its content being that exported rows").build();
-
-    @Override
-    public void init(ProcessorInitializationContext context) {
-        List<PropertyDescriptor> list = new ArrayList<>();
-        list.add(DATABASE_CLIENT_SERVICE);
-        list.add(PLAN);
-        list.add(MIMETYPE);
-        properties = Collections.unmodifiableList(list);
-
-        Set<Relationship> set = new HashSet<>();
-        set.add(FAILURE);
-        set.add(SUCCESS);
-        relationships = Collections.unmodifiableSet(set);
-    }
-
-    @Override
-    public void onTrigger(ProcessContext context, ProcessSessionFactory sessionFactory) throws ProcessException {
-        final ProcessSession session = sessionFactory.createSession();
-
-        try {
-            FlowFile flowFile = session.get();
-            if (flowFile == null) {
-                flowFile = session.create();
+	public static final PropertyDescriptor PLAN = new PropertyDescriptor.Builder()
```

Comment:
@mariusmarklogic Make sure your editor is picking up the .editorconfig file in the root of this project - we want to avoid a bunch of changes like this solely based on whitespace changes. Makes it more difficult to see what actually changed. 

If your editor doesn't support editorconfig files, that's fine - just make sure it has the same whitespace settings as what's in the .editorconfig file.

Reply:
i think now it is working